tests/unit/dummyImage.ts

```diff
@@ -0,0 +1,4 @@
+// https://gist.github.com/ondrek/7413434?permalink_comment_id=3914686#gistcomment-3914686
+const dummyImage = `data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAgAAAAIAQMAAAD+wSzIAAAABlBMVEX///+/v7+jQ3Y5AAAADklEQVQI12P4AIX8EAgALgAD/aNpbtEAAAAASUVORK5CYII`;
+
+export default dummyImage;
```

Comment:
typos.tomlに書く必要が出てきてしまうのであれば、ここは不正確でも良いので簡潔なものにしても良いかもと思いました。
こんな感じとか。
```suggestion
const dummyImage = `data:image/png;base64,DUMMY`;
export default dummyImage;
```

Reply:
意味のある文字列にして、typosに書かないようにしました。